.github/workflows/hitl.yml

```
@@ -1,4 +1,4 @@
-name: HITL
+name: HITL Run Request
```

Maybe

```suggestion
name: HITL Trigger
```

And assuming the filename defines the bare "hitl" in https://github.com/quartiq/stabilizer/runs/2305293775, can re rename it to `.github/workflows/hitl_trigger.yml`?